NEWS.md

```diff
@@ -1,3 +1,11 @@
+# rlistings 0.1.2.0
+ * `cols` argument renamed to `disp_cols` in `as_listing`
```

Comment:
Hi @gmbecker , I wonder if we could use "display_cols" rather than "disp_cols", so it is more readable? 
@Melkiades @edelarua @ayogasekaram @telepath37 @clarkliming , any thoughts?

Reply:
I support renaming

Reply:
I just have no preference either is fine